docs/sources/staged.rst

```diff
@@ -11,6 +11,7 @@ Supported content types:
 * RPMs
 * Advisories
 * Generic files
+* Content Gateway
```

Comment:
I think it should mention that we are only dealing with metadata or definitions for CGW, like:

```suggestion
* Content Gateway metadata
```